src/zenml/models/v2/base/filter.py

```diff
@@ -248,31 +246,23 @@ def generate_query_conditions_from_column(self, column: Any) -> Any:
 
         if self.operation == GenericFilterOps.ONEOF:
             assert isinstance(self.value, list)
-            # Convert the list of values to a list of json strings
-            json_list = (
-                [json.dumps(v) for v in self.value]
-                if self.json_encode_value
-                else self.value
-            )
-            return column.in_(json_list)
+            return column.in_(self.value)
```

Comment:
`oneof` IMO should have the same logic as `equals`, just for multiple entries. The logic for `equals` however changed as part of this PR, so I think it should also be applied here?

Reply:
So, are you saying that we should add an `equals` condition (that we used down below) for each entry in the list and then combine them with an `or`?

Reply:
Yes

Reply:
Adressed this as well.